app/core/base.js

```diff
@@ -9,9 +9,11 @@
 // WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 // License for the specific language governing permissions and limitations under
 // the License.
-
+import "babel-polyfill";
```

Comment:
I need this to get test running I think promise implementation was missing I will try see I can remove requiring the whole polyfill